Gruntfile.js

```diff
@@ -4,7 +4,65 @@ module.exports = function (grunt) {
 
     require('time-grunt')(grunt);
 
+    var derequire = require('derequire');
+    var envify = require('envify/custom');
+
+    var envifyTransform = envify({NODE_UNIQUE_ID: 0});
+
+    // String to be prepended to source files
+    var banner = "/*! shortId.js\n https://github.com/dylang/shortid\n Copyright (c) Dylan Greene\n License: MIT +no-false-attribs License\n*/\n";
+
     grunt.initConfig({
+        browserify: {
+            dist: {
+                src: ['lib/shortid.js'],
+                dest: 'shortId.js'
```

Comment:
how about writing all files to a `/dist/` dir. Also lets just use lowercase for the filenames so a typo doesn't just happen to work on Macs but fails when pushed to a Linux server.